src/list-grid.scss

```diff
@@ -263,13 +265,19 @@ $block: #{$fd-namespace}-grid-list;
 
   &__filter,
   &__filter-button {
-    color: #fff !important;
+    color: var(--sapHighlightTextColor) !important;
   }
 
   &__filter-button {
     @include fd-hover() {
       color: var(--sapButton_TextColor) !important;
     }
+
+    @include fd-active() {
+      @include fd-hover() {
+        color: var(--sapHighlightTextColor) !important;
```

Comment:
have you tried without `!important`?

Reply:
yes, i tried without !important, which does not work. because we have already `!important` applied on `__filter-button` color. and that is necessary because transparent `fd-button` has text colored.

Reply:
That's why specificity exists in CSS. You can avoid using !important just by nesting the css classes and start relying on specificity. :

```
  .#{$block}__filter {
    &[class*="active"] {
      @include fd-fiori-focus();

      color: var(--sapHighlightTextColor);
    }
  }

  .#{$block}__filter-button {
    color: var(--sapHighlightTextColor);

    @include fd-hover() {
      color: var(--sapButton_TextColor);
    }

    @include fd-active() {
      @include fd-hover() {
        color: var(--sapHighlightTextColor);
      }
    }
  }

```

Reply:
using specificity to remove `!important`. working fine.
thanks.